bugsnag-android-core/src/test/java/com/bugsnag/android/EventStateTest.java

```diff
@@ -0,0 +1,45 @@
+package com.bugsnag.android;
+
+import static com.bugsnag.android.BugsnagTestUtils.convert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Collections;
+
+public class EventStateTest {
+
+    private HandledState handledState
+            = HandledState.newInstance(HandledState.REASON_HANDLED_EXCEPTION);
+    private ImmutableConfig config;
+    private Event event;
+
+    /**
+     * Generates a new default event for use by tests
+     */
+    @Before
+    public void setUp() {
+        Configuration configuration = BugsnagTestUtils.generateConfiguration();
+        this.config = convert(configuration);
+        RuntimeException exception = new RuntimeException("Example message");
+        event = new Event(exception, config, handledState);
+    }
+
+    @Test
+    public void shouldIgnoreMatches() {
```

Comment:
Test for null/empty configuration option has not be included in refactor.

Reply:
I have added an additional test case for an empty `Set` - null will not be a valid value after #674